### DISCUSSION TITLE
How is glasskube different from helm from package management standpoint?

### DISCUSSION THREAD

**Discussion by murindwaz** [glasskube/glasskube#183] [Q&A] [answered]
How is glasskube different from helm from package management standpoint?

**Comment by pmig** [comment, marked as answer]
Hi @murindwaz,

We dedicated a page on our website to the comparison to helm: https://glasskube.dev/docs/comparisons/helm/

_tldr:_ we are not a full replacement of helm but aim to be the easiest method for Kubernetes administrators to install packages and keep them up to date.


Do you have any more specific or detailed questions?
